libminifi/src/Site2SiteClientProtocol.cpp

```diff
@@ -38,7 +39,8 @@ namespace minifi {
 
 bool Site2SiteClientProtocol::establish() {
   if (_peerState != IDLE) {
-    logger_->log_error("Site2Site peer state is not idle while try to establish");
+    logger_->log_error(
```

Comment:
OK, i will change the format, i want to reduce the line length to make it easy to read